src/main/java/io/javalin/Javalin.java

```diff
@@ -210,27 +307,55 @@ private void ensureActionIsPerformedBeforeServerStart(@NotNull String action) {
 
     // End embedded server methods
 
+    /**
+     * Defines an access manager for the instance. Secured endpoints require one to be set.
+     *
+     * @see <a href="https://javalin.io/documentation#access-manager">Access manager in docs</a>
+     */
     public Javalin accessManager(@NotNull AccessManager accessManager) {
         this.accessManager = accessManager;
         return this;
     }
 
+    /**
+     * Handles exceptions thrown in the handlers.
+     *
+     * @see <a href="https://javalin.io/documentation#exception-mapping">Exception mapping in docs</a>
+     */
     public <T extends Exception> Javalin exception(@NotNull Class<T> exceptionClass, @NotNull ExceptionHandler<? super T> exceptionHandler) {
         exceptionMapper.getExceptionMap().put(exceptionClass, (ExceptionHandler<Exception>) exceptionHandler);
         return this;
     }
 
+    /**
+     * Adds a lifecycle event listener.
+     *
+     * The method must be called before {@link Javalin#start()}.
+     *
+     * @see <a href="https://javalin.io/documentation#lifecycle-events">Events in docs</a>
+     */
     public Javalin event(@NotNull EventType eventType, @NotNull EventListener eventListener) {
         ensureActionIsPerformedBeforeServerStart("Event-mapping");
         eventManager.getListenerMap().get(eventType).add(eventListener);
         return this;
     }
 
+    /**
+     * Handles error cases in handlers based on status code
+     *
+     * @see <a href="https://javalin.io/documentation#error-mapping">Error mapping in docs</a>
+     */
     public Javalin error(int statusCode, @NotNull ErrorHandler errorHandler) {
         errorMapper.getErrorHandlerMap().put(statusCode, errorHandler);
         return this;
     }
 
+    /**
+     * Accepts all requests defined by this handler group.
```

Comment:
Hmm... "Creates a scope with a temporary static instance so that you can call static methods on `ApiBuilder` without referencing the instance" ? Needs to be worded better 🤔

Reply:
Should it really mention static instance and all the stuff?
I suppose it could be: 

> Adds a group of handlers defined by `ApiBuilder` static methods.

We omit implementation details and bring mention of `ApiBuilder` and its methods.